src/app/feedback/feedback.directive.ts

```diff
@@ -146,20 +148,24 @@ export class FeedbackDirective {
 
   @HostListener('click')
   openFeedback() {
-    const title = $localize`Feedback`;
-    const type = 'feedback';
-    const fields = dialogFieldOptions;
-    const formGroup = {
-      priority: [ this.priority, Validators.required ],
-      type: [ this.type, Validators.required ],
-      message: [ this.message, CustomValidators.required ]
-    };
-    this.dialogsFormService
-      .confirm(title, fields, formGroup)
-      .pipe(debug('Dialog confirm'))
-      .subscribe((response) => {
-        if (response !== undefined) {
-          this.addFeedback(response);
+    this.authService.isAuthenticated().subscribe(isAuthenticated => {
```

Comment:
It might be a little more readable or quickly understandable that we're checking authentication here if the listener function is a `checkAuthentication` and we move the open feedback logic to a separate function that is called here.

I'm thinking something like:

```
@HostListener('click')
checkAuthentication() {
  this.authService.isAuthenticated().subscribe(this.openFeedback); // might need to be () => this.openFeedback(); for "this" context
}

openFeedback() {
  ...
}
```